setup.py

```diff
@@ -12,6 +12,7 @@
 
 "The Qiskit Terra setup file."
 
+import json
```

Comment:
Don't think we use json here any more.

Reply:
Done in: https://github.com/Qiskit/qiskit-terra/pull/10530/commits/453c815d992bae937ab7a2a3b2f484ab36d26e67